backend/src/main/kotlin/com/tngtech/apicenter/backend/config/ContentNegotiationConfiguration.kt

```diff
@@ -0,0 +1,23 @@
+package com.tngtech.apicenter.backend.config
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.dataformat.yaml.YAMLMapper
+import com.fasterxml.jackson.module.kotlin.KotlinModule
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.web.servlet.config.annotation.ContentNegotiationConfigurer
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer
+import org.springframework.http.MediaType
+
+@Configuration
+class WebConfig : WebMvcConfigurer {
+
+  override fun configureContentNegotiation(configurer: ContentNegotiationConfigurer) {
+    configurer.
+            parameterName("mediaType").
+            ignoreAcceptHeader(true).
```

Comment:
Why ignore the header? This is preferable to a query paramter, in my opinion.

Reply:
This is required... when this line is deleted, there is a 406 error when requesting content other than JSON. Fraid I really can't explain why that happens.

Reply:
Okay, this is sensible after the discussion that the formats of the resource and of the content can be different. But maybe we should rename the parameter to something more inconspicuous like "downloadFormat".